### PR TITLE
Access admin dashboards using ingress + basic auth

### PR DESCRIPTION
**Pull request by mkjpryor**

### CODE REVIEW

File: roles/admin_dashboard_ingress/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    else undef(hint = 'admin_dashboard_ingress_tls_wildcard_key is required')
+  }}
+#   The TLS secret name when a wildcard certificate is used
+admin_dashboard_ingress_tls_wildcard_secret_name: admin-dashboard-tls
+
+
+# The basic auth username to use for admin dashboards
+admin_dashboard_ingress_basic_auth_username: admin
+# The basic auth password to use for admin dashboards
+admin_dashboard_ingress_basic_auth_password: >-
```
Comment (JohnGarbutt) [inline roles/admin_dashboard_ingress/defaults/main.yml:40]: People are going to need to add this on upgrade, maybe we need to call this a breaking change?

**Review by JohnGarbutt — APPROVED** [review]
OK, that is much simpler than I expected, nice.

File: roles/admin_dashboard_ingress/defaults/main.yml (hunk 1 above)
Comment (mkjpryor) [reply roles/admin_dashboard_ingress/defaults/main.yml:40]: Yes. The release notes stuff I rolled out everywhere specifically respects a `breaking` label that can be added to PRs to make them appear right at the top in their own section

**Review by sd109 — APPROVED** [review]
LGTM

**Pull request merged**